types/astring/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for astring 1.3
+// Project: https://github.com/davidbonnet/astring
+// Definitions by: Nikolaj Kappler <https://github.com/nkappler>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as ESTree from 'estree';
+import "node";
```

Comment:
Why?

Reply:
the import "stream" in the next line is part of Node and is not resolved otherwise.
Another solution that might work is to add a package.json and add node as a dependency but that might be overkill.
I am open for suggestions on how to import it but I couldn't figure out another way that worked.